concent_api/core/tests/test_payments_bankster.py

```diff
@@ -256,208 +257,327 @@ def test_that_when_there_are_other_deposit_claims_finalize_payment_substract_the
+        self.requestor_deposit_account.save()
+
+    def create_deposit_claim(self, amount, tx_hash):
+        deposit_claim = DepositClaim(
+            payee_ethereum_address=self.task_to_compute.provider_ethereum_address,
+            payer_deposit_account=self.requestor_deposit_account,
+            amount=amount,
+            concent_use_case=ConcentUseCase.FORCED_PAYMENT,
+            tx_hash=tx_hash,
+            closure_time=parse_timestamp_to_utc_datetime(get_current_utc_timestamp()),
+        )
+        deposit_claim.full_clean()
+        deposit_claim.save()
+
+    def call_settle_overdue_acceptances_with_mocked_sci_functions(
+        self,
+        get_deposit_value_return_value=None,
+        get_list_of_payments_return_value=None,
+    ):
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch('core.payments.bankster.validate_list_of_transaction_timestamp') as self.validate_list_of_transaction_mock:
+                with mock.patch(
+                    'core.payments.bankster.service.get_deposit_value',
+                    return_value=(
+                        get_deposit_value_return_value if get_deposit_value_return_value is not None
+                        else self.get_deposit_value_return_value_default
+                    ),
+                ) as self.get_deposit_value_mock:
+                    with mock.patch(
+                        'core.payments.bankster.service.get_list_of_payments',
+                        return_value=(
+                            get_list_of_payments_return_value if get_list_of_payments_return_value is not None
+                            else self._get_list_of_force_transactions()
+                        ),
+                    ) as self.get_list_of_payments_mock:
+                        claim_against_requestor = settle_overdue_acceptances(
+                            requestor_ethereum_address=self.task_to_compute.requestor_ethereum_address,
+                            provider_ethereum_address=self.task_to_compute.provider_ethereum_address,
+                            acceptances=self.subtask_results_accepted_list,
+                            requestor_public_key=hex_to_bytes_convert(self.task_to_compute.requestor_public_key),
+                        )
+
+        return claim_against_requestor
+
+    def assert_mocked_function_were_called(self):
+        self.get_deposit_value_mock.assert_called_once()
+        self.get_list_of_payments_mock.assert_called_once()
+        self.validate_list_of_transaction_mock.assert_called_once()
+
+    def test_that_settle_overdue_acceptances_should_return_none_if_subtask_costs_where_already_paid(self):
+        """
+        In this test we have following calculations:
+
+        TaskToCompute price:                                    3000
+        Sum of amounts from list of forced transactions:        3000
+        Requestor deposit value:                                does not matter in this case
```

Comment:
I would suggest stating the value anyway, ie:
```suggestion
         Requestor deposit value:                                15000 (does not matter in this case)
```

Reply:
:+1: done